upup/pkg/fi/nodeup/nodetasks/package.go

```diff
@@ -326,27 +329,18 @@ func (_ *Package) RenderLocal(t *local.LocalTarget, a, e, changes *Package) erro
 			env = append(env, "DEBIAN_FRONTEND=noninteractive")
 		}
 
-		if d.IsDebianFamily() {
-			// Each time apt-get install is run, the timestamp of /var/lib/dpkg/status is updated.
-			// To avoid unnecessary apt-get update calls, we check the timestamp of this file.
-			// If it is newer than 10 minutes, we skip apt-get update.
-			stat, err := os.Stat("/var/lib/dpkg/status")
+		// If the package manager update was less than 10 minutes ago, skip updating the package list.
+		if d.IsDebianFamily() && time.Since(packageManagerLastUpdated) > 10*time.Minute {
```

Comment:
Why 10 min ? I think `apt update` take less than 5 min to run.

Reply:
This is meant to run `apt-get update` every 10 min (arbitrary number). I expect bootstrap to finish much faster, but seems reasonable.